api/src/main/scala/hmda/api/http/institutions/submissions/SubmissionSignPaths.scala

```diff
@@ -85,8 +93,58 @@ trait SubmissionSignPaths
 
     onComplete(fSubmission) {
       case Success(sub) =>
+        if (sub.status.code == 10) {
```

Comment:
The front end requests the `/sign` endpoint multiple times, which calls the `completeWithSubmissionReceipt()` method.  The status would only not be `Signed` if the `fSign` Future didn't complete successfully, in which case we return an error.

That being said, I didn't account for a user returning to this page _after_ a signature.  I think that might trigger another email, so I'm switching up my method of detecting if an email is needed.

Reply:
Do they really request the `POST` method of the `/sign` endpoint multiple times? I know they request the `GET` method to view the receipt, but if they're doing the `POST` more than once, that sounds like a bug

Reply:
> The status would only not be Signed if the fSign Future didn't complete successfully, in which case we return an error.

In that case, why is the if statement necessary?

Reply:
(Sorry for my erroneous comments. I'm reading more closely before I comment again 😬 )

Reply:
Looks like the call to `emailSignature` should perhaps move to line 70 of this file, so it can only get triggered by the `POST` method of this endpoint.

Reply:
Both the `GET` and `POST` endpoints call the same method, `completedWithSubmissionReceipt()`.  I call the `emailSignature()` method from here because of the convenience of already having the `Submission` object, which includes fields necessary for the email.

However, from within the `completedWithSubmissionReceipt()`, there is no way of telling whether the user is requesting the `GET` or `POST` endpoint.  I remedied this by checking if the submission was signed, which would only happen after a successful call to the `POST` endpoint.  

My previous approach had a potential bug (detailed above), which I'm fixing soon in a new commit.

Reply:
Thanks for the explanation. The fix looks 👍